sesman/chansrv/Makefile.am

```diff
@@ -63,7 +65,8 @@ xrdp_chansrv_SOURCES = \
   sound.c \
   sound.h \
   xcommon.c \
-  xcommon.h
+  xcommon.h \
+  config.c
```

Comment:
This file list is ordered, so config.c should be added in another place. BTW the name config.c could be better renamed to reflext what it does